eng/common/pipelines/templates/steps/bypass-local-dns.yml

```diff
@@ -0,0 +1,7 @@
+parameters:
+    VmImage: ''
+steps:
+    # https://github.com/actions/virtual-environments/issues/798
+    - script: sudo ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
+      displayName: Bypass local DNS server to workaround issue resolving cognitiveservices names
+      condition: and(succeededOrFailed(), eq(variables['VmImage'], 'ubuntu-18.04'))
```

Comment:
@praveenkuttappan, should the variable be VmImage or OSVmImage? The variable being inspected in [eng/pipelines/templates/jobs/archetype-sdk-tests.yml](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L81) is OSVmImage.

Reply:
I have updated the name.